dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCPV.mo

```diff
@@ -22,50 +22,57 @@ model SVarCPV "PV static var compensator model"
   parameter Types.PerUnit BMinPu "Minimum value for the variable susceptance in p.u (base UNom, SnRef)";
-  parameter Types.PerUnit LambdaPu "Statism of the regulation law URefPu = UPu + LambdaPu*QPu in p.u (base UNom, SnRef)";
   parameter Types.PerUnit BShuntPu "Fixed susceptance of the static var compensator in p.u (for standby mode) (base UNom, SnRef)";
   parameter Types.VoltageModule UNom "Static var compensator nominal voltage in kV";
+  parameter Types.VoltageModulePu URef0Pu "Start value of the voltage regulation set point in p.u (base UNom)";
 
-  Modelica.Blocks.Interfaces.RealInput URef(start = URef0) "Voltage reference for the regulation in kV";
-  Modelica.Blocks.Interfaces.BooleanInput selectModeAuto(start = selectModeAuto0) "Whether the static var compensator is in automatic configuration";
-  Modelica.Blocks.Interfaces.IntegerInput setModeManual(start = setModeManual0) "Mode selected when in manual configuration";
+  input Types.VoltageModule URef(start = URef0Pu * UNom) "Voltage reference for the regulation in kV";
+  input Boolean selectModeAuto(start = selectModeAuto0) "Whether the static var compensator is in automatic configuration";
+  input Integer setModeManual(start = setModeManual0) "Mode selected when in manual configuration";
 
-  Types.PerUnit BVarRawPu(start = BVar0Pu) "Raw variable susceptance of the static var compensator in p.u (base UNom, SnRef)";
   Types.PerUnit BVarPu(start = BVar0Pu) "Variable susceptance of the static var compensator in p.u (base UNom, SnRef)";
   Types.PerUnit BPu(start = B0Pu) "Susceptance of the static var compensator in p.u (base UNom, SnRef)";
   Types.VoltageModulePu UPu(start = U0Pu) "Voltage amplitude at terminal in p.u (base UNom)";
   Types.ReactivePowerPu QInjPu(start = B0Pu * U0Pu ^ 2) "Reactive power in p.u (base SnRef) (generator convention)";
   Types.ActivePowerPu PInjPu(start = 0) "Active power in p.u (base SnRef) (generator convention)";
-  Types.VoltageModulePu URefPu(start = URef0 / UNom) = modeHandling.URefPu "Reference voltage amplitude in p.u (base UNom)";
+  Types.VoltageModulePu URefPu(start = URef0Pu) = modeHandling.URefPu "Reference voltage amplitude in p.u (base UNom)";
 
-  BaseControls.ModeHandling modeHandling(Mode0 = Mode0, UNom = UNom, URefDown = URefDown, URefUp = URefUp, UThresholdDown = UThresholdDown, UThresholdUp = UThresholdUp, tThresholdDown = tThresholdDown, tThresholdUp = tThresholdUp, URef0 = URef0);
+  BaseControls.ModeHandling modeHandling(Mode0 = Mode0, UNom = UNom, URefDown = URefDown, URefUp = URefUp, UThresholdDown = UThresholdDown, UThresholdUp = UThresholdUp, tThresholdDown = tThresholdDown, tThresholdUp = tThresholdUp, URef0 = URef0Pu * UNom);
 
 protected
   parameter Types.PerUnit B0Pu "Start value of the susceptance in p.u (base UNom, SnRef)";
   parameter Types.VoltageModulePu U0Pu  "Start value of voltage amplitude at injector terminal in p.u (base UNom)";
```

Comment:
One space and not two before comment (sorry, I think it was my mistake) (also for other parameters)